src/models/bullet.js

```diff
@@ -0,0 +1,47 @@
+import {CONFIG} from '../config'
+import {WORLD} from '../world'
+import {applyVector} from '../util'
+import {Circle} from './base'
+
+const {colors} = CONFIG
+const {yellow, white} = colors
+
+const _bullet = ({x, y, radius, vector, speed, owner}) => {
+  let _x = x
+  let _y = y
+  let _radius = radius
+  let _vector = vector
+  let _speed = speed
+  let _owner = owner
```

Comment:
Not a fan of this structure, though I think it's temporary in lieu of eventual `mobx`ery?

Reply:
Yea; I don't like it either -- I really want mobx at this point, but I wanna get the big missing pieces in first because I'm worried mobx'll be a time suck.